admin/src/templates/release.html

```diff
@@ -1,8 +1,11 @@
 <div class="col-xs-5">
     <span class="label label-info release-name" title="{{release | buildVersion}}">{{release | buildVersion}}</span>
```

Comment:
I chose not to edit this, so the branch name will still look like `12312-foo-bar (~ 5.0.0)` because our upgrade e2e test logic looks at this, and then it's a mess when older branches try to run this e2e test and it becomes a selector mess.

Reply:
Well, in the selector for the branch name, we intentionally include the parenthesis, but nothing after the parenthesis as far as I can tell. So we expect the branch name to looks something like `branch_name\s(.*`. 

https://github.com/medic/cht-core/blob/b9ddc064d82158b65cd801eb8488ccbab4838de0/tests/page-objects/upgrade/upgrade.wdio.page.js#L13

So a bit of a mess. 
The issue is that for long branch names the base version might be truncated :(

Reply:
I guess I am still not following with what the trouble here is. (Sorry!)  Looking at that line of code, it seems like we could just drop the ` (` part from the selector and it would just work (or at least work as well as it did before with regards to the truncated branch names...).  Bonus points if we add an assertion for the new base version column.